src/status_im/contexts/profile/settings/events.cljs

```diff
@@ -71,13 +71,14 @@
                                        {:on-success on-success}])
              :on-cancel nil}]]})))
 
-(rf/reg-event-fx :profile.settings/toggle-goerli-test-network
+(rf/reg-event-fx
+ :profile.settings/toggle-goerli-test-network
  (fn [{:keys [db]}]
-   (let [value      (get-in db [:profile/profile :is-goerli-enabled?])
+   (let [value      (get-in db [:profile/profile :goerli-enabled?])
          on-success #(rf/dispatch [:logout])]
      {:fx [[:ui/show-confirmation
             {:content   (i18n/label :t/goerli-testnet-toggle-confirmation)
-             :on-accept #(rf/dispatch [:profile.settings/profile-update :is-goerli-enabled?
+             :on-accept #(rf/dispatch [:profile.settings/profile-update :goerli-enabled?
```

Comment:
@smohamedjavid thanks for the reference!

Ok, I guess the "react field name" should have been `goerli-enabled?` instead of `is-goerli-enabled?` :cry: 
We should keep it as before then.

Fortunately, I expected a reason like this, so it's easy to rollback these changes.

Reply:
Done!

Reply:
I agree @ulisesmac. I know it's frustrating, we have to make changes in status-go first and make changes in clients' apps in one shot.

Additionally, renaming the DB column (altering the table) is not supported in iOS. We have to create a new column (`goerli_enabled`) and use it.

There should be a guideline for these things (naming variables and such) in `status-go` just as we have for `mobile`.

Reply:
100% @smohamedjavid - that would be great to see if it doesn't already exist!